crates/oxc_linter/src/snapshots/typescript_consistent_type_definitions.snap

```diff
@@ -147,3 +147,31 @@ snapshot_kind: text
  3 │               foo: string;
    ╰────
   help: Use an `type` instead of a `interface`
+
+  ⚠ typescript-eslint(consistent-type-definitions): Use an `interface` instead of a `type`
+   ╭─[consistent_type_definitions.tsx:1:10]
+ 1 │ declaretype S={}
+   ·        ────
```

Comment:
The underline offset error is unrelated to this PR and seems to be a diagnostic issue with displaying errors for special characters.